backend/typescript/models/camp.model.ts

```diff
@@ -35,134 +35,139 @@ export interface Camp extends Document {
+      type: String,
+    },
+    dropoffFee: {
+      type: Number,
+      required: true,
+    },
+    pickupFee: {
+      type: Number,
+      required: true,
+    },
+    endTime: {
+      type: String,
+      required: true,
+    },
+    earlyDropoff: {
+      type: String,
+    },
+    fee: {
+      type: Number,
+      required: true,
+    },
+    fileName: {
+      type: String,
+    },
+    formQuestions: {
+      type: [
+        {
+          type: Schema.Types.ObjectId,
+          ref: "FormQuestion",
+        },
+      ],
+      default: [],
+      required: true,
+    },
+    latePickup: {
+      type: String,
+    },
+    location: {
+      streetAddress1: {
+        type: String,
+        required: true,
       },
-    ],
-    default: [],
-  },
-  campCounsellors: {
-    type: [
-      {
-        type: Schema.Types.ObjectId,
-        ref: "User",
+      streetAddress2: {
+        type: String,
+        required: false,
       },
-    ],
-    default: [],
-  },
-  campSessions: {
-    type: [
-      {
-        type: Schema.Types.ObjectId,
-        ref: "CampSession",
+      city: {
+        type: String,
+        required: true,
       },
-    ],
-    default: [],
-  },
-  description: {
-    type: String,
-  },
-  dropoffFee: {
-    type: Number,
-    required: true,
-  },
-  pickupFee: {
-    type: Number,
-    required: true,
-  },
-  endTime: {
-    type: String,
-    required: true,
-  },
-  earlyDropoff: {
-    type: String,
-  },
-  fee: {
-    type: Number,
-    required: true,
-  },
-  fileName: {
-    type: String,
-  },
-  formQuestions: {
-    type: [
-      {
-        type: Schema.Types.ObjectId,
-        ref: "FormQuestion",
+      province: {
+        type: String,
+        required: true,
       },
-    ],
-    default: [],
-    required: true,
-  },
-  latePickup: {
-    type: String,
-  },
-  location: {
-    streetAddress1: {
+      postalCode: {
+        type: String,
+        required: true,
+      },
+    },
+    name: {
       type: String,
       required: true,
     },
-    streetAddress2: {
+    campProductId: {
       type: String,
-      required: false,
     },
-    city: {
+    dropoffPriceId: {
       type: String,
-      required: true,
     },
-    province: {
+    dropoffProductId: {
+      type: String,
+    },
+    pickupPriceId: {
+      type: String,
+    },
+    pickupProductId: {
       type: String,
-      required: true,
     },
-    postalCode: {
+    startTime: {
       type: String,
       required: true,
     },
+    volunteers: {
+      type: String,
+    },
   },
-  name: {
-    type: String,
-    required: true,
-  },
-  campProductId: {
-    type: String,
-  },
-  dropoffPriceId: {
-    type: String,
-  },
-  dropoffProductId: {
-    type: String,
-  },
-  pickupPriceId: {
-    type: String,
-  },
-  pickupProductId: {
-    type: String,
-  },
-  startTime: {
-    type: String,
-    required: true,
-  },
-  volunteers: {
-    type: String,
-  },
-});
+  { timestamps: true },
```

Comment:
Does this correlate to the `createdAt` and `updatedAt` fields above? Also those fields are set to string not date. is that ok?

Reply:
Updated it to Date based on https://codesource.io/learn-how-to-use-mongoose-timestamps/